docs/user/subprojects.rst

```diff
@@ -1,46 +1,74 @@
+but still be maintained independently.
 
 This is useful for:
 
 * Organizations that need all their projects visible in one documentation portal or landing page
 * Projects that document and release several packages or extensions
 * Organizations or projects that want to have a common search function for several sets of documentation
 
-For example, a parent project, ``foo`` is set up with a subproject, ``bar``. The
-documentation for ``foo`` will be available at:
+For example, a parent project, ``example-project`` is set up with a subproject, ``plugin``. The
+documentation for ``example-project`` will be available at:
 
-https://foo.readthedocs.io/en/latest/
+https://example-project.readthedocs.io/en/latest/
 
-The documentation for ``bar`` will be available under this same path:
+The documentation for ``plugin`` will be available under this same path:
 
-https://foo.readthedocs.io/projects/bar/en/latest/
+https://example-project.readthedocs.io/projects/plugin/en/latest/
 
 .. seealso::
 
    :doc:`/guides/subprojects`
 
-
 Sharing a custom domain
 -----------------------
 
-Projects and subprojects can also be used to share a custom domain with a number
-of projects. To configure this, one project should be established as the parent
-project. This project will be configured with a custom domain. Projects can then
-be added as subprojects to this parent project.
+Projects and subprojects can be used to share a custom domain.
+To configure this, one project should be established as the main project.
+This project will be configured with a custom domain.
+Other projects are then added as subprojects to the main project.
 
 If the example project ``foo`` was set up with a custom domain,
 ``docs.example.com``, the URLs for projects ``foo`` and ``bar`` would
-respectively be at: 
+respectively be at:
 
 * `foo`: https://docs.example.com/en/latest/
 * `bar`: https://docs.example.com/projects/bar/en/latest/
 
+.. note::
+
+   The terms "parent" and "child" are also used,
+   where the main project is the parent and all subprojects are a child.
+   You can **not** have a child of a child,
+   meaning that you cannot add a subproject to a subproject.
+
+Using aliases
+-------------
+
+You can choose an alias for the subproject when it is created.
+This allows you to override the URL that is used to access it,
+giving more control over how you want to structure your projects.
+
+You can set your subproject's project name and :term:`slug` however you want,
+but we suggest prefixing it with the name of the main project.
+
+Typically, a subproject is created with a ``<mainproject>-`` prefix,
+for instance if the main project is called ``foo`` and the subproject is called ``bar``,
+then the subproject's Read the Docs project name will be ``foo-bar``.
+When adding the subproject,
+the alias is set to ``bar`` and the project's URL becomes
+``foo.readthedocs.io/projects/bar``.
```

Comment:
This is a really clear explanation 👍 I do wonder about using real examples here too, instead of foo/bar, but it's less important I think.

Reply:
The use of example project names is consistent now. The foos and the bars are gone.